tests/model_hub_tests/torch_tests/test_hf_transformers.py

```diff
@@ -125,6 +125,39 @@ def load_model(self, name, type):
             preprocessor = CLIPFeatureExtractor.from_pretrained(name)
             encoded_input = preprocessor(self.image, return_tensors='pt')
             example = dict(encoded_input)
+        elif 'xclip' in mi.tags:
+            from transformers import XCLIPVisionModel
+            
+            model = XCLIPVisionModel.from_pretrained(name)
+            example = (torch.randn(*(16, 3, 224, 224), dtype=torch.float32), )  # needs video as input
+        elif 'audio-spectrogram-transformer' in mi.tags:
+            example = (torch.randn(*(1, 1024, 128), dtype=torch.float32), )
+        elif 'mega' in mi.tags:
+            from transformers import AutoModel
+            
+            model = AutoModel.from_pretrained(name)
+            example = model.dummy_inputs
+            example.update({'output_attentions': True, 'output_hidden_states': True, 'return_dict': True})
```

Comment:
Is this really required? will it work without this?

Reply:
without this flags, there are `None`s on outputs and model becomes untraceable
![image](https://github.com/openvinotoolkit/openvino/assets/5703039/aaec706b-c3ca-487f-b3ac-473e805f8568)

Reply:
is it possible to use  model config for manipulate with this values?
e.g. model.config.return_dict = True

Reply:
done: replaced this with setting config